onnx/onnxifi.h

```diff
@@ -493,6 +506,30 @@ typedef int32_t onnxBackendInfo;
  */
 #define ONNXIFI_OPTIMIZATION_LOW_DELAY 3
 
+/**
+ * Log events which caused a failure in an ONNXIFI function call.
```

Comment:
Does this line belong to `ONNXIFI_LOG_LEVEL_ERROR `? For `ONNXIFI_LOG_LEVEL_DEBUG` and `ONNXIFI_LOG_LEVEL_INFO ` they don't necessary correspond to some failures.

Reply:
Good catch!

Reply:
Yes, it is for `ONNXIFI_LOG_LEVEL_ERROR`. `ONNXIFI_LOG_LEVEL_INFO` and `ONNXIFI_LOG_LEVEL_DEBUG` also enable logging of errors, but can log many additional events too.